crates/ruff_db/src/system.rs

```diff
@@ -54,6 +54,11 @@ pub trait System {
     /// Returns the current working directory
     fn current_directory(&self) -> &SystemPath;
 
+    fn read_dir<'a>(
```

Comment:
Can we rename the method to read_directory. We use directory everywhere else

Reply:
I called it `read_dir()` for consistency with `std::fs::read_dir()`, `camino::Utf8Path::read_dir()` and `camino::Utf8Path::read_dir_utf8()`. But I guess our equivalent of e.g. `is_dir()` in those other libraries is `is_directory()` (and I prefer fully spelling it out as well). So I'll make the change!